autogen/agentchat/contrib/postgresql_agent.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+import json
+import re
+
+import psycopg as pg
+from psycopg.rows import dict_row
+
+from textwrap import dedent
+from typing import Any, Callable, Dict, List, Optional, Literal, Optional, Union
+
+from autogen import ConversableAgent
+from autogen.coding import CodeExecutor, CodeExtractor, MarkdownCodeExtractor, CodeBlock, CodeResult
+from autogen.runtime_logging import log_new_agent, logging_enabled
+
+class PostgreSqlQueryExecutor(CodeExecutor):
+    def __init__(self, dsn: str, **kwargs):
+        """(Experimental) A code executor class that executes SQL SELECT query statement.
+
+        **This will execute SQL SELECT query statement on PostgreSQL server.**
+
+        The code blocks are executed or save in the order they are received.
```

Comment:
```suggestion
        The code blocks are executed or saved in the order they are received.
```

Reply:
I'll remove it instead since the sql will only be executed, never saved:
`The code blocks are executed in the order they are received.`

Is that alright?